goaviatrix/security_domain.go

```diff
@@ -69,6 +69,9 @@ func (c *Client) CreateSecurityDomain(securityDomain *SecurityDomain) error {
 		return errors.New("Json Decode add_route_domain failed: " + err.Error() + "\n Body: " + bodyString)
 	}
 	if !data.Return {
+		if strings.Contains(data.Reason, "already exist") {
+			return nil
```

Comment:
This is error. The user should be importing in this scenario. Someone else could be managing it

Reply:
Make sense.